topicExtractor.py

```diff
@@ -243,6 +243,11 @@ def retrieveTopics(config, videoData, overwrite=False):
 
     logging.info("Generating & saving Topic Model and Data...")
 
+    if videoData is None:
+        logging.error("Video Data not provided. Exiting...")
+        sys.exit("Video Data not provided. Exiting...")
```

Comment:
Optional: I don't feel the logging.error or sys.exit needs same messaging once should be good enough. what is the purpose for it? since the logs might convey what's going at that point of the script

if you have to use same text then making the text as variable and use in both cases. Same goes to other places in the code here

Reply:
I will resolve this in the next PR.

Reply:
This has been addressed in the final commit.